{{cookiecutter.project_name}}/README.md

```diff
@@ -17,6 +17,16 @@ You will need:
 - `docker` with [version at least](https://docs.docker.com/compose/compose-file/#compose-and-docker-compatibility-matrix) `18.02`
 
 
+## Hot start
```

Comment:
We do not encourage "hot starts". We encourage people to read the whole docs.

There are actually more steps to take before starting the projects:
0. Understanding the workflow
1. Getting the correct configuration
2. Applying migrations
3. Seeding the database
4. Compiling translations

But, this information is actually helpful to the newcomers. We can incorporate it with this section: https://wemake-django-template.readthedocs.io/en/latest/_pages/template/dependencies.html#troubleshooting

And also you can extend https://wemake-django-template.readthedocs.io/en/latest/_pages/template/configuration.html#secret-settings to be more precise. How to get the correct `.env` values? Where to look?

And you can also describe some specific problems in "Troubleshooting" section on the "Configuration" page. Which is missing right now.

Reply:
I would be more than happy to look up the mentioned sections, but I do insist on adding hot start section, because there are other developers like me, who would prefer (or will have to) launch the project first.

Reply:
Ok, maybe we can add links to the most important parts of the existing docs to this new section?

Something like this:

```
## Hot start

Before starting the project, you will need to read these documents:

- Link to getting the secret configuration
- Link to running docker
- Link to etc
```

Otherwise people **will** ignore other steps. I know people. They do not like to read more than they need to.